data/strings/types_strings.txt

```diff
@@ -12850,17 +12855,93 @@
     comment = Hiking trail tagged as sac_scale=demanding_mountain_hiking (3 of 6) or trail_visibility=bad.
     ref = type.highway.path
     en = Difficult or Indistinct Trail
+    af = Moeilike of swak sigbare roete
+    ar = درب صعب أو ضعيف الرؤية
+    az = Çətin və ya zəif görünən cığır
+    be = Складаная ці дрэнна бачная сцежка
+    bg = Трудна или слабо видима пътека
+    ca = Ruta difícil o poc visible
+    cs = Obtížná nebo špatně viditelná stezka
+    da = Vanskelig eller dårligt synlig sti
     de = Schwieriger oder kaum erkennbarer Pfad
+    el = Δύσκολη ή ελάχιστα ορατή διαδρομή
+    es = Sendero difícil o poco visible
+    et = Raske või halvasti nähtav rada
+    eu = Ibilbide zaila edo gaizki ikusten da
+    fa = مسیر دشوار یا ضعیف قابل مشاهده است
+    fi = Vaikea tai huonosti näkyvä reitti
+    fr = Sentier difficile ou peu visible
+    he = שביל קשה או לא נראה לעין
+    hi = कठिन या ख़राब दिखाई देने वाला मार्ग
+    hu = Nehéz vagy rosszul látható nyomvonal
+    id = Jejak yang sulit atau kurang terlihat
     it = Sentiero difficile o poco tracciato
+    ja = 困難な、または見通しの悪いトレイル
+    ko = 어렵거나 잘 보이지 않는 트레일
+    lt = Sudėtingas arba blogai matomas takas
+    mr = अवघड किंवा खराब दृश्यमान पायवाट
+    nb = Vanskelig eller dårlig synlig sti
+    nl = Moeilijk of slecht zichtbaar pad
+    pl = Trudny lub słabo widoczny szlak
+    pt = Percurso difícil ou pouco visível
+    pt-BR = Trilha difícil ou pouco visível
+    ro = Traseu dificil sau puțin vizibil
     ru = Сложная или плохо видимая тропа
+    sk = Náročná alebo zle viditeľná trasa
+    sv = Svår eller dåligt synlig led
+    sw = Njia ngumu au isiyoonekana vizuri
+    th = เส้นทางที่ยากลำบากหรือมองเห็นได้ไม่ดี
+    tr = Zor veya kötü görünür patika
+    uk = Складна або погано видима стежка
+    vi = Đường đi khó hoặc khó nhìn thấy
+    zh-Hans = 难走或难见的小路
+    zh-Hant = 困難或不明顯的小路
 
   [type.highway.path.expert]
     comment = Hiking trail tagged as sac_scale=alpine_hiking (4+ of 6) or trail_visibility=horrible or more extreme.
     ref = type.highway.path
-    en = Expert or Indiscernible Trail
+    en = Expert or indistinguishable Trail
```

Comment:
```suggestion
    en = Expert or pathless Trail
```

I'm not a native speaker, but does "indistinguishable" really make sense her? "Indiscernible" or "pathless" are better IMO.

Reply:
pathless sounds strange. Indiscernible makes most sense IMO, but indistinguishable works too

Reply:
Pathless comes from the [Wiki](https://wiki.openstreetmap.org/wiki/Key:trail_visibility). But yes, a pathless path is to some degree an oxymoron.